src/index.html

```diff
@@ -7,12 +7,73 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image-container">
+        <img
+          src="./images/imac.jpeg"
+          alt="item card"
+          class="card__image"
+        />
+      </div>
+
+      <div class="card__name">
+        <div class="card__name-main">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__name-product">Product code: 195434</div>
+      </div>
+
+      <div class="card__review">
+        <div class="card__review-stars" aria-label="four stars">
+          <div class="card__review-stars-star"></div>
+
+          <div class="card__review-stars-star"></div>
+
+          <div class="card__review-stars-star"></div>
+
+          <div class="card__review-stars-star"></div>
+
+          <div class="card__review-stars-star"></div>
+        </div>
+
+        <div class="card__review-review">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-text">Price:</div>
+        <div class="card__price-cost">$2,199</div>
+      </div>
+
+      <button
```

Comment:
The button lacks a descriptive `aria-label` attribute, which is important for accessibility. Consider adding it.